### PR TITLE
Dilpreet | Stage | Integrate thermal pdf task on live for check usb connector issue

### PR DESCRIPTION
**Pull request by dilpreet77**
https://app.clickup.com/t/1ke3z9

### CODE REVIEW

**Review by ravinderwalkover — APPROVED** [review]

File: apps/web-giddh/src/app/invoice/thermal-pdf/thermal-pdf.component.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<strong>Status: {{ status }}</strong>
+<div>
+    <input [(ngModel)]="ip" type="text" name="ip" placeholder="IP of printer with WebPRNT" />
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/thermal-pdf/thermal-pdf.component.html:3]: Do use translation variables throughout the file

File: apps/web-giddh/src/app/invoice/thermal-pdf/thermal-pdf.component.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        this.printService.isConnected.subscribe(result => {
+            this.status = result;
+            if (result) {
+                console.log('Connected to printer!!!');
+            } else {
+                console.log('Not connected to printer.');
+            }
+        });
+    }
+    public  ngOnInit(): void {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/thermal-pdf/thermal-pdf.component.ts:26]: Avoid empty lifecycle hooks

File: apps/web-giddh/src/app/invoice/thermal-pdf/thermal-pdf.component.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    public status: boolean = false;
+    public usbPrintDriver: UsbDriver;
+    public webPrintDriver: WebPrintDriver;
+    public ip: string = '';
+
+    constructor(private printService: PrintService) {
+        this.usbPrintDriver = new UsbDriver();
+        this.printService.isConnected.subscribe(result => {
+            this.status = result;
+            if (result) {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/thermal-pdf/thermal-pdf.component.ts:19]: Avoid console logs

**Review by prateek-giddh — CHANGES_REQUESTED** [review]
Do use branch name based on convention: https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App#contributing-guide

**Pull request closed without merging**